lib/mongoid/findable.rb

```diff
@@ -116,6 +116,36 @@ def exists?(id_or_conditions = :none)
       with_default_scope.exists?(id_or_conditions)
     end
 
+    # Return true if any documents exist in the criteria.
+    #
+    # @example Determine if any documents exist
+    #   criteria.any?
+    #   
+    # @return [ true | false ] If any documents exist.
+    def any?(*args, &block)
```

Comment:
It's safe to leave off the `*args` and `&block` here, since they aren't used in the method body anymore.

Ditto for the other two methods you added. 👍